04-stl-basics/tasks/03-createSortedList.cpp

```diff
@@ -3,6 +3,15 @@
 
 // Implement createSortedList
 // add proper include :)
+ list <int> createSortedList(vector<int> vec){
+ list <int> myList;
+
+    for (size_t i =0; i<vec.size(); i++){
+    myList.push_back(vec[i]);
+     }
+myList.sort();
+return myList;
+}
```

Comment:
Twoje formatowanie kodu jest nieczytelne. Formatuj w ten sposób:
```suggestion
list<int> createSortedList(vector<int> vec) {
    list<int> myList;

    for (size_t i = 0; i < vec.size(); i++) {
        myList.push_back(vec[i]);
    }
    myList.sort();
    return myList;
}
```